app_feup/test_report.json

```diff
@@ -0,0 +1 @@
oCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:5"},"result":{"status":"passed","duration":5011000000}},{"keyword":"And ","name":"I am logged in","line":6,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:6"},"result":{"status":"passed","duration":5000000}},{"keyword":"And ","name":"I open the drawer","line":7,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:7"},"result":{"status":"passed","duration":2410000000}},{"keyword":"And ","name":"I tap the button that contains the text \"Escolha de Turmas\"","line":8,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:8"},"result":{"status":"passed","duration":1351000000}},{"keyword":"Given ","name":"I don't have courses selected for class registration","line":18,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:18"},"result":{"status":"passed","duration":4951000000}},{"keyword":"When ","name":"I tap the element that contains the text \"Nenhuma unidade curricular\"","line":19,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:19"},"result":{"status":"passed","duration":418000000}},{"keyword":"And ","name":"I tap the first child of type \"CheckboxListTile\" of the widget of type \"ListView\"","line":20,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:20"},"result":{"status":"passed","duration":1190000000}},{"keyword":"And ","name":"I tap the first child of type \"CheckboxListTile\" of the widget of type \"ListView\"","line":21,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:21"},"result":{"status":"passed","duration":925000000}},{"keyword":"And ","name":"I tap the back button","line":22,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:22"},"result":{"status":"passed","duration":404000000}},{"keyword":"Then ","name":"I expect the text \"Nenhuma unidade curricular\" to be present","line":23,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:23"},"result":{"status":"passed","duration":24000000}}]}]}]
```

Comment:
I'd add this file to .gitignore and remove it.

Reply:
I wanted to make the file available for the teacher to see, but you're right, it makes no sense for it to be here. I'm gonna bring it over to the other repository and add it to the .gitignore here.